src/views/product-tutorials-view/components/sitemap/sitemap.module.css

```diff
@@ -4,6 +4,7 @@
 
 .heading {
   composes: hds-typography-display-300 from global;
+  composes: g-offset-scroll-margin-top from global;
```

Comment:
The `Sitemap`'s heading appears in the table of contents, so it needs `g-offset-scroll-margin-top`.